Package.swift

```diff
@@ -23,30 +23,24 @@ let swiftSettings: [SwiftSetting] = [
 
 let package = Package(
     name: "swift-openapi-runtime",
-    platforms: [
-        .macOS(.v10_15), .macCatalyst(.v13), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .visionOS(.v1)
-    ],
-    products: [
-        .library(
-            name: "OpenAPIRuntime",
-            targets: ["OpenAPIRuntime"]
-        )
-    ],
-    dependencies: [
-        .package(url: "https://github.com/apple/swift-http-types", from: "1.0.0"),
-    ],
+    platforms: [.macOS(.v10_15), .macCatalyst(.v13), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .visionOS(.v1)],
+    products: [.library(name: "OpenAPIRuntime", targets: ["OpenAPIRuntime"])],
+    dependencies: [.package(url: "https://github.com/apple/swift-http-types", from: "1.0.0")],
     targets: [
         .target(
             name: "OpenAPIRuntime",
-            dependencies: [
-                .product(name: "HTTPTypes", package: "swift-http-types")
-            ],
-            swiftSettings: swiftSettings
-        ),
-        .testTarget(
-            name: "OpenAPIRuntimeTests",
-            dependencies: ["OpenAPIRuntime"],
+            dependencies: [.product(name: "HTTPTypes", package: "swift-http-types")],
             swiftSettings: swiftSettings
-        ),
+        ), .testTarget(name: "OpenAPIRuntimeTests", dependencies: ["OpenAPIRuntime"], swiftSettings: swiftSettings),
     ]
 )
+
+// ---    STANDARD CROSS-REPO SETTINGS DO NOT EDIT   --- //
```

Comment:
Are we missing a START: here?

Reply:
This is to try and have the lengths of the two blocks the same and have the text centered so it isn’t too ugly. We can revisit the exact structure of this the next time the common block is updated